Gems/Atom/Feature/Common/Assets/Materials/Special/DebugVertexStreams.azsl

```diff
@@ -12,9 +12,16 @@
 #include <Atom/RPI/ShaderResourceGroups/DefaultDrawSrg.azsli>
 #include <Atom/RPI/TangentSpace.azsli>
 
+// These enums need to be kept in sync with the enum values in DebugVertexStreams_IncompatibleEnums.lua
+option enum class DebugVertexStream { Normals, Tangents, Bitangents, Uvs, TangentW } o_debugVertexStream = DebugVertexStream::Normals;
+option enum class TangentOptions { UseVertexData, UseSurfaceGradient} o_tangentOptions = TangentOptions::UseVertexData;
+option enum class BitangentOptions { UseVertexData, UseSurfaceGradient, ReconstructBitangent} o_bitangentOptions = BitangentOptions::UseVertexData;
+option enum class ColorDisplayMode {ColorSpace, UnitSpace} o_colorDisplayMode = ColorDisplayMode::ColorSpace;
 
-option enum class DebugVertexStream { Normals, Tangents, Bitangents, UVs, TangentW } o_debugVertexStream = DebugVertexStream::Normals;
-option enum class ReconstructionMethods { UseVertexData, UseSurfaceGradient, ReconstructBitangent} o_reconstructionMethod = ReconstructionMethods::UseVertexData;
+ShaderResourceGroup MaterialSrg : SRG_PerMaterial
+{
+    uint m_uvIndex;
```

Comment:
This could be a shader option as well since it's uniform.

Reply:
They could really all be options or all be in the srg. Since this is not a performance intensive material and is for debug visualization purposes. I just started with what I was familiar with (options) and followed StandardPbr for an example of how to use the uv enum, so I ended up with the split.

I'm going to leave it for now.